src/Controller/Link/Create.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Link;
+
+use App\Entity\Link;
+use App\Repository\LinkRepositoryInterface;
+use App\Service\RandomString\GeneratorInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+class Create
+{
+    private LinkRepositoryInterface $repository;
+    private GeneratorInterface $generator;
+
+    public function __construct(LinkRepositoryInterface $repository, GeneratorInterface $generator)
+    {
+        $this->repository = $repository;
+        $this->generator = $generator;
+    }
```

Comment:
То же самое, просто подключить rector и использовать последние возможности языка.
```suggestion
       public function __construct(
        private LinkRepositoryInterface $repository,
        private GeneratorInterface      $generator)
    {
    }
```